swan/scripts/before-notebook.d/02_environment.sh

```diff
@@ -130,6 +130,28 @@ if [ "$SOFTWARE_SOURCE" == "lcg" ]; then
   fi
 
   _log "Finished setting up CVMFS and user environment"
+else
+  # Configure NXCALS for customenvs
+  export SPARK_HOME=/cvmfs/sft.cern.ch/lcg/releases/spark/3.5.1-dfe72/x86_64-el9-gcc13-opt
+
+  # Configure necessary settings (it doesn't run without this)
+  NBCONFIG=$JPY_DIR/nbconfig
+  mkdir -p $NBCONFIG
+  LOCAL_NB_NBEXTENSIONS=$SWAN_PROJECTS/.notebook_nbextensions
+  if [ ! -f $LOCAL_NB_NBEXTENSIONS ]; then 
+    echo "{
+      \"load_extensions\": {
+      }
+    }" > $LOCAL_NB_NBEXTENSIONS
+  fi
+  rm -f $NBCONFIG/notebook.json
+  ln -s $LOCAL_NB_NBEXTENSIONS $NBCONFIG/notebook.json
+
+  export KRB5CCNAME=$KRB5CCNAME_NB_TERM
```

Comment:
This needs to be in a separate commit, with a message that explains that we need to set this variable here to point KRB5CCNAME to a writable directory. This is done for LCG already but not for custom envs, that's why we need it.